### ISSUE TITLE
SnakeViz integration

### ISSUE THREAD

**Issue by AgDude** [django-silk/silk#83]
Does silk actually perform profiling? Or only log sql queries?

This is a nice looking project, and didn't take long to set up, but I didn't see anything I would call a profile, and don't see any screenshots of one in the readme. There is a nice log of all database queries, but when I hear about a profile in python, I think of something like this:
https://docs.python.org/2/library/profile.html

**Comment by AgDude** [comment]
I decided to use http://jiffyclub.github.io/snakeviz/. If silk could be combine with snakeviz, so silk put the profile in the database, then somehow had a link for snakeviz to view it, that would be awesome. Clearly this would not be a trivial undertaking, snakeviz wants to read a file, and doesn't know anything about django models.

**Comment by mtford90** [comment]
@AgDude yep silk's profiling is limited to profiling time taken to execute blocks of code & num. database queries executed in that block of code and allows you to trace back those db queries to their origin. @JoshData contributed a cProfile integration a while back though, which sounds more like what you're wanting. 

If you fancy contributing the snakeviz integration I'll happy add you to the django-silk org - this would be an awesome enhancement & i'd be around for any questions you may have wrt. the code base. Doubt I'll have time to implement this myself though - which you can probably tell by my awful response time :(

**Comment by hanleyhansen** [comment]
When I add `SILKY_PYTHON_PROFILER = True` to my settings files I get a nice cProfile output in the Profiling tab under Python Profiling. 

I copied that text and saved it in a `.prof` but when I run it through snakeviz i get the following error:

```bash
snakeviz: error: the file ~/Desktop/test1.prof is not a valid profile. Generate profiles using:
python -m cProfile -o my_program.prof my_program.py
```

Any ideas?

**Comment by avelis** [comment]
@AgDude @hanleyhansen Was enhancement possibly pushed out in a recent release?

**Comment by hanleyhansen** [comment]
Yes. The PR i submitted adds a download button to the Profiling page where you can download the binary .prof file and use it in snakeviz. 

See: https://github.com/django-silk/silk/pull/123

That was merged and uploaded to pypi.

**Comment by avelis** [comment]
@hanleyhansen Thanks for confirming with me. I will close this issue and say it was released in v0.6.0 as an exportable feature.
